messages/de.json

```diff
@@ -3,18 +3,19 @@
 	"navbar": {
 		"upload": "Hochladen",
 		"convert": "Konvertieren",
-		"settings": "Optionen",
+		"settings": "Einstellungen",
```

Comment:
to be fair i'm not german, but is it a big issue if this was kept as "Optionen" instead? this translation may be better but it does cause overflow

<img width="196" height="96" alt="Image" src="https://github.com/user-attachments/assets/80d56131-39b8-43f9-b0b1-26e88ae41336" />